code/datums/trading/unique.dm

```diff
@@ -25,6 +25,64 @@
 /datum/trader/ship/unique/what_do_you_want()
 	return get_response("what_want", "I don't want anything!")
 
+/datum/trader/ship/unique/syndicate
+	name = "Cyndie Kate"
+	origin = "Cloaked ship"
+
+	possible_wanted_items = list (
+		/obj/item/weapon/gun/energy/captain					= TRADER_THIS_TYPE,
+		/obj/item/weapon/hand_tele							= TRADER_THIS_TYPE,
+		/obj/item/blueprints								= TRADER_THIS_TYPE,
+		/obj/item/weapon/disk/nuclear						= TRADER_THIS_TYPE,
```

Comment:
Have you tested if they can take the nuke disk with them on the shuttle?
Afaik we have restrictions for that on some shuttles.

Reply:
Tested - good question, didn't think of it. Works without issue.